config/initializers/assets.rb

```diff
@@ -8,4 +8,4 @@
 
 # Precompile additional assets.
 # application.js, application.css, and all non-JS/CSS in app/assets folder are already added.
-Rails.application.config.assets.precompile += %w(favicons/* small-flag.png)
+Rails.application.config.assets.precompile += %w(misc/saml-post.js)
```

Comment:
My understanding is that it needs to be pre-compiled so that it's discoverable / referenced via `javascript_include_tag`